coffee/classes/mds_markdown.coffee

```diff
@@ -93,6 +93,7 @@ module.exports = class MdsMarkdown
   constructor: (settings) ->
     opts         = extend({}, MdsMarkdown.default.options, settings?.options || {})
     plugins      = extend({}, MdsMarkdown.default.plugins, settings?.plugins || {})
+    twemoji.base = settings?.twemojiBase || twemoji.base
```

Comment:
Well probably the base-path won't change, but I think it should not set as twemoji's common setting when it has per `MdsMarkdown` instances.

And `MdsMarkdown` class has its `@default` variable for putting default options. I wish to set local path as a `@default.twemoji.base`.